src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const price = rental * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days < 7) {
+    return price - 20;
+  }
+
+  if (days >= 7) {
+    return price - 50;
```

Comment:
Instead of using literals for the discount values and the minimum number of days to qualify for discounts, use constants with descriptive names. This will make the code more readable and maintainable, as per checklist item 2.